specs/core/0_beacon-chain.md

```diff
@@ -174,6 +168,7 @@ The following values are (non-configurable) constants used throughout the specif
 
 | Name | Value |
 | - | - |
+| `COMMITTEES_PER_SLOT` | `2**5` (= 32) |
 | `SHARD_COUNT` | `2**10` (= 1,024) |
```

Comment:
Remove?

Reply:
Will do. There are a few things still in there just beacuse phase 1 uses them and the build fails if I take them out...